phac-nml/staramr
# WIP:Feature/restore default databases

This pull request adds in the ability to restore a database build error back to the valid default (issue #3).

To test, you can run:

```
staramr db update -d
```

This should fail to build the database.

You will then get error messages if you try to use this database:

```
staramr db info
Error with default database [data/update]. Please try restoring with 'staramr db restore'
```

```
staramr search -o out *.fasta
Default database [data/update] is in an error state. Please try restoring with 'staramr db restore'.
```

You can restore with:

```
staramr db restore
```

Now `info` and `search` should work properly.
